scripts/linters/pylint_extensions.py

```diff
@@ -1857,6 +1857,33 @@ def process_tokens(self, tokens):
                         args=(token),
                         line=line_num)
 
+class InequalityWithNoneChecker(checkers.BaseChecker):
+    """Custom pylint checker prohibiting use of "if x != None" and
+    enforcing use of "if x is not None" instead.
+    """
+    __implements__ = interfaces.ITokenChecker
```

Comment:
Hi @michaelw54, This looks great! I wanted to check whether we can implement this using the IAstroidChecker? [We try using IToken only for the comments related checks.]